docs/javascript/quickstart.md

```diff
@@ -0,0 +1,683 @@
+---
+title: Ionic JavaScript Quickstart
+sidebar_label: Quickstart
+---
+
+<head>
+  <title>Ionic JavaScript Quickstart: JavaScript Basics</title>
+  <meta
+    name="description"
+    content="Ionic JavaScript Quickstart covers the basics of JavaScript and Ionic, including any Ionic-specific features. Learn how to build JavaScript apps."
+  />
+</head>
+
+import DocsCard from '@components/global/DocsCard';
+import DocsCards from '@components/global/DocsCards';
+
+Welcome! This guide will walk you through the basics of Ionic JavaScript development using Vite. You'll learn how to set up your development environment, create a simple project, explore the project structure, and understand how Ionic components work. This is perfect for getting familiar with Ionic JavaScript before building your first real app.
+
+If you're looking for a high-level overview of what Ionic JavaScript is and how it fits into the web development ecosystem, see the [Ionic JavaScript Overview](overview).
```

Comment:
Nitpick, some people may use a screen reader or other ways to access content. It would be great to use neutral words.

```suggestion
If you're looking for a high-level overview of what Ionic JavaScript is and how it fits into the web development ecosystem, review the [Ionic JavaScript Overview](overview).
```